optimade/validator/validator.py

```diff
@@ -113,10 +120,12 @@ def get(self, request: str):
                 the `MAX_RETRIES` attempts.
 
         """
-        if request:
-            self.last_request = f"{self.base_url}{request}".replace("\n", "")
+        if urllib.parse.urlparse(request, allow_fragments=True).scheme:
```

Comment:
Do you still want to make sure the `scheme` is part of `("http", "https")` or do we not care? If yes, here's a quick and dirty suggestion:
```suggestion
        request_scheme = urllib.parse.urlparse(request, allow_fragments=True).scheme
        if request_scheme and request_scheme in ("http", "https"):
```
Otherwise, fine 👍

Reply:
I'm going to leave it as is, on the off-chance that an implementation provides a different scheme (presumably accidentally) then it will be obvious from the error spat out. Your suggested change would lead to the next request being e.g. `http://base_url.com/ssh://base_url.com?page_offset=5`, so I think it's clearer as is.

Reply:
Sure. Then the docstring should probably have been updated, but fine.
Again, I just want to be really nitpicky with everything in the validator since (in my opinion) it should be usable for every and any implementation. I.e., it should be really specific in what it wants, while at the same time being very broad in scope.. not easy :)